tests/integration/test_manager.py

```diff
@@ -231,15 +235,17 @@ async def test_finish_job(get_all, pg_job_manager, job_factory):
     assert started_at.date() == datetime.datetime.utcnow().date()
     assert await get_all("procrastinate_jobs", "attempts") == [{"attempts": 0}]
 
-    await pg_job_manager.finish_job(job=job, status=jobs.Status.SUCCEEDED)
+    await pg_job_manager.finish_job(
+        job=job, status=jobs.Status.SUCCEEDED, delete_job=False
+    )
     expected = [{"status": "succeeded", "attempts": 1}]
     assert await get_all("procrastinate_jobs", "status", "attempts") == expected
 
 
 async def test_finish_job_retry(get_all, pg_job_manager, job_factory):
     await pg_job_manager.defer_job_async(job_factory())
     job1 = await pg_job_manager.fetch_job(queues=None)
-    await pg_job_manager.finish_job(job=job1, status=jobs.Status.TODO)
+    await pg_job_manager.finish_job(job=job1, status=jobs.Status.TODO, delete_job=False)
 
     job2 = await pg_job_manager.fetch_job(queues=None)
```

Comment:
I think this test was actually meaningless, even before your PR. We should have a look at it, in another PR.